libimagtodo/src/delete.rs

```diff
@@ -0,0 +1,31 @@
+// Needed for reading a Json File
+// extern crate rustc_serialize;
+// use rustc_serialize::json::Json;
+// use std::fs::File;
+// use std::io::Read;
+
+use std::ops::Deref;
+use toml::Value;
+
+use libimagstore::store::Store;
+use libimagstore::storeid::IntoStoreId;
+use module_path::ModuleEntryPath;
+
+/// With the uuid we get the storeid and than we can delete the entry
+fn deleteFunc(uuid: i32, store : &Store) {	
```

Comment:
Using the `Uuid` type from the crate `uuid` (already in Cargo.toml) might be better. See https://doc.rust-lang.org/uuid/uuid/struct.Uuid.html for documentation on how to format it.